Cargo.toml

```diff
@@ -134,9 +134,7 @@ tide = "^0.16.0"
 # Including brotli *very* slow, so don't do that. Including the "default" feature pulls a mime-type list from the internet on build, which isn't used.
 tide-compress = { version="0.10.6", default-features = false, features = [ "gzip", "regex-check" ] }
 tide-openssl = "^0.1.1"
-# Unable to increase version due to removing ability to detect
-# local platform time.
-time = "=0.2.27"
+time = { version = "^0.3.0", features = ["formatting", "local-offset"] }
```

Comment:
Needs to be 0.3.19 else a hidden cfg param is needed

Reply:
Nice of them to be incredibly vague about this in the changelogs :rage:

Done!